chain/client/Cargo.toml

```diff
@@ -55,10 +56,11 @@ expensive_tests = []
 adversarial = ["near-network/adversarial", "near-chain/adversarial"]
 metric_recorder = ["near-client-primitives/metric_recorder"]
 delay_detector = ["near-chain/delay_detector", "near-network/delay_detector", "delay-detector"]
-protocol_feature_block_header_v3 = ["near-primitives/protocol_feature_block_header_v3", "near-chain/protocol_feature_block_header_v3", "near-store/protocol_feature_block_header_v3"]
+protocol_feature_block_header_v3 = ["near-primitives/protocol_feature_block_header_v3", "near-chain/protocol_feature_block_header_v3", "near-store/protocol_feature_block_header_v3", "nearcore/protocol_feature_block_header_v3"]
```

Comment:
Yeah, I tried to apply some hack to test `near-client` separately: https://github.com/near/nearcore/issues/4351#issuecomment-855140473
But it doesn't work anyway because `neard` is not building with it